### PR TITLE
chore: native rendering datasource information.

### PR DESCRIPTION
**Pull request by LukaszParala**

### CODE REVIEW

**Review by pnowicki-auctane — APPROVED** [review]

File: native-rendering/documents-designer/data-source.md [hunk 1]
```diff
@@ -14,6 +14,13 @@ or the bracket notation:
 ```code
 $['shipment']['tracking_number']
 ```
+
+For each document you need to select DataSource property. Select a 'package' if you want to print label for each package, and 'shipment' if you want only one label for whole shipment.
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/data-source.md:18]:
```suggestion
For each document you need to select DataSource property value:
* **package** -  prints a label for each package in the shipment;
* **shipment** - prints only one label for the whole shipment.
```
To consider, and check whether formats and looks nice. Bold values are consistent with listings on  DesignerWindow page

File: native-rendering/documents-designer/data-source.md [hunk 2]
```diff
@@ -83,7 +90,7 @@ $.shipment.metadata['account.number']
 
 #### Current package reference
 
-You can also reference to the current *Package* using prefix `@`
+If you select 'package' as DataSources, you can also reference to the current *Package* using prefix `@`
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/data-source.md:93]:
```suggestion
If you select **package** as DataSource, you can also refer to the current *Package* using prefix `@`
```
At least 'DataSources' typo to be fixed

**Review by PrzemyslawSpringer — COMMENTED** [review]
cosmetics

File: native-rendering/documents-designer/data-source.md [hunk 3]
```diff
@@ -14,6 +14,15 @@ or the bracket notation:
 ```code
 $['shipment']['tracking_number']
 ```
+
+For each document you need to select DataSource property value:
+**package** -  prints a label for each package in the shipment;
+**shipment** - prints only one label for the whole shipment.
```
Comment (PrzemyslawSpringer) [inline native-rendering/documents-designer/data-source.md:20]:
```suggestion
* **package** -  prints a label for each package in the shipment;
* **shipment** - prints only one label for the whole shipment.
```
You lost punctuation

**Pull request merged**